pandas/io/sql.py

```diff
@@ -1079,7 +1079,8 @@ def _sqlalchemy_type(self, col):
 
         return Text
 
-    def _get_dtype(self, sqltype):
+    @staticmethod
```

Comment:
-1 on static methods generally; you can move to a free function if you really want

Reply:
I agree we should generally be using staticmethods only sparingly, but IMO here that is fine. This method is only used once, just above, and keeping it here keeps it closer to where it is used. 
The same for `_fetchall_as_list`. And the `get_table` is actually overriding a parent class method, so shouldn't be moved to a free function.

So I prefer to keep those functions where they are. Whether they are normal methods or staticmethods, I personally don't care. So if others don't want to use staticmethods, I would just leave this as is.

Reply:
I guess with the -1 and with the leave as is, I will close this PR.